target/linux/sunxi/Makefile

```diff
@@ -14,7 +14,7 @@ FEATURES:=fpu usb ext4 display rtc squashfs
 SUBTARGETS:=cortexa8 cortexa7 cortexa53
 MAINTAINER:=Zoltan HERPAI <wigyori@uid0.hu>
 
-KERNEL_PATCHVER:=4.14
+KERNEL_PATCHVER:=4.19
```

Comment:
Please don't do this, it's probably going to be changed for all targets at once 19.03 is branched.